src/index.ts

```diff
@@ -319,14 +305,11 @@ function launchMetals(
           const location =
             params.arguments && (params.arguments[0] as Location);
           if (location) {
-            const range = Range.create(
```

Comment:
The `metals-goto-location` stuff that you removed here will need to stay to answer those requests. I don't believe it's tree view specific?

Reply:
Metals sends `metals-goto-location` command when user opens selected class or trait from tree view 'metalsBuild'.

I don't like line
```
workspace.selectRange(range);
```
because it left vim in visual mode.